modules/ROOT/pages/peoplesoft-connector.adoc

```diff
@@ -28,7 +28,7 @@ Because of how closely the connector is coupled with PeopleSoft, this document a
 [NOTE]
 The definition and detailed explanation of the PeopleSoft terms and concepts used here are freely available in the PeopleBooks documentation set of your PeopleSoft installation.
 
-This document assumes you are familiar with Mule, xref:anypoint-connectors.adoc[Anypoint Connectors], and xref:6@studio::index.adoc[Anypoint Studio Essentials]. To increase your familiarity with Studio, consider completing one or more xref:general:getting-started:anypoint-connector.adoc[Anypoint Connector quickstart]. Further, this page assumes that you have a basic understanding of xref:elements-in-a-mule-flow.adoc[Mule flows] andxref:global-elements.adoc[Mule Global Elements].
+This document assumes you are familiar with Mule, xref:anypoint-connectors.adoc[Anypoint Connectors], and xref:6@studio::index.adoc[Anypoint Studio Essentials]. Further, this page assumes that you have a basic understanding of xref:elements-in-a-mule-flow.adoc[Mule flows] andxref:global-elements.adoc[Mule Global Elements].
```

Comment:
We don't need to use the version in the xref unless this is intentionally xreffing Studio 6 instead of 7 (which it might be).